Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -1302,7 +1314,7 @@ localizedStrings["Press %s to start running the audit."] = "Press %s to start ru
 localizedStrings["Press %s to stop editing audits."] = "Press %s to stop editing audits.";
 localizedStrings["Press %s to stop running."] = "Press %s to stop running.";
 localizedStrings["Pressed"] = "Pressed";
-localizedStrings["Click to pretty print"] = "Click to pretty print";
+localizedStrings["Pretty print"] = "Pretty print";
```

Comment:
this seems out of topic.

Reply:
This is just a result of running `update-webkit-localizable-strings`; it likely caught string changes or sorted differently.